src/app/components/JumpTo/index.tsx

```diff
@@ -0,0 +1,61 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import useViewTracker from '#app/hooks/useViewTracker';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import { EventTrackingMetadata } from '#app/models/types/eventTracking';
+import Text from '#app/components/Text';
+import InlineLink from '#app/components/InlineLink';
+import idSanitiser from '../../lib/utilities/idSanitiser';
+
+export interface JumpToProps {
+  jumpToHeadings?: Array<{ heading: string }>;
+  eventTrackingData?: EventTrackingMetadata;
+}
+const JumpTo = ({ jumpToHeadings, eventTrackingData }: JumpToProps) => {
+  const { translations } = useContext(ServiceContext);
+  const { jumpTo = 'Jump to' } = translations?.articlePage || {};
+
+  const viewRef = useViewTracker(eventTrackingData);
+  const clickTrackerHandler = useClickTrackerHandler({
+    ...eventTrackingData,
+    componentName: 'jumpto',
+  });
+
+  const headingId = 'jump-to-heading';
+
+  // we use the Text component with the as prop set to strong (for now) because the screenreader UX states the heading should not be announced
+  // using InlineLink instead of anchor to bring benefits to styling, but can revert to anchor if needed
+  return (
+    <section
```

Comment:
Worth double checking the a11y on this. Also the `tabIndex={-1}` on the `<Text>` component which renders the 'Jump To' header.

Other than that, its looks great!